seahorn/trusty/user/base/lib/tipc/proof/tipc_send1/main.c

```diff
@@ -0,0 +1,36 @@
+#include <seahorn/seahorn.h>
+#include <lib/tipc/tipc.h>
+#include <trusty_ipc.h>
+#include <sea_tipc_helper.h>
+
+extern handle_t nd_handle(void);
+extern int nd_size_t(void);
+extern void *sea_malloc_havoc(size_t sz);
```

Comment:
I don't think this function signature is recognized by seahorn op sem; you would still need to implement the actual function `sea_malloc_havoc`, or copy the existing from aws c common project

Reply:
I guess I have not ported these. They have to be part of our libc port